clients/privacy-center/components/Card.tsx

```diff
@@ -20,6 +20,8 @@ const Card: React.FC<CardProps> = ({
     bg="white"
     py={8}
     px={6}
+    border="1px solid"
+    borderColor="gray.200"
```

Comment:
Recent figma designs all have a small border around the card at all times for some contrast against the light background.